### PR TITLE
Prs13205 yohannes

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: new/PRS13205Yohannes.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ዮሐንስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"><roleName type="title">ṗāṗās</roleName>Yoḥannǝs</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic">"bishop of Ethiopia"</occupation>
```
Comment (DariaElagina) [inline new/PRS13205Yohannes.xml:51]: the quotation marks are not needed

File: new/PRS13205Yohannes.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ዮሐንስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"><roleName type="title">ṗāṗās</roleName>Yoḥannǝs</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic">bishop of Ethiopia</occupation>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
```
Comment (thea-m) [inline new/PRS13205Yohannes.xml:54]: They are not wrong, but it is better to delete these empty elements to have a cleaner file.

File: new/PRS13205Yohannes.xml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                    <floruit>He is an author of a homily on Mary, in which
```
Comment (thea-m) [inline new/PRS13205Yohannes.xml:55]: "the author"?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

**Review by thea-m — APPROVED** [review]

**Pull request merged**